reposerver/repository/repository.go

```diff
@@ -318,6 +319,22 @@ func (s *Service) runRepoOperation(
 			return err
 		}
 		defer io.Close(closer)
+		if !s.initConstants.AllowOutOfBoundsSymlinks {
+			err := argopath.CheckOutOfBoundsSymlinks(chartPath)
```

Comment:
just cosmetic change , i think it would be nice move this block to some dedicate function, so you will not need duplicate it fully in bottom

Reply:
Gonna go ahead and merge this, and @notfromstatefarm can follow up with another PR if this can be DRY'ed out.